Sources/tart/Commands/Run.swift

```diff
@@ -64,6 +65,10 @@ struct Run: AsyncParsableCommand {
                                               """, valueName: "interface name"))
   var netBridged: String?
 
+  @Flag(help: ArgumentHelp("Use software networking instead of the default shared (NAT) networking",
+          discussion: "Learn how to configure Softnet for use with Tart here: https://github.com/cirruslabs/softnet"))
+  var netSoftnet: Bool = false
+
   func validate() throws {
```

Comment:
Maybe just make `netSoftnet = netSoftnet  || withSoftnet` to respect the old value and then change all occurrences of `withSoftnet` to `netSoftnet`?

Reply:
I don't think that (2) it will be hard to remove that and (2) that we should remove it because `validate()`'s errors are based on the specific flag used.